scipy/integrate/quadrature.py

```diff
@@ -853,3 +859,105 @@ def newton_cotes(rn, equal=0):
     fac = power*math.log(N) - gammaln(p1)
     fac = math.exp(fac)
     return ai, BN*fac
+
+
+def gh_quad(func, n, args=(), mu_hat=0, sigma_hat=1, zero_nan=False):
+    """
+    Gauss-Hermite quadrature
+
+    Compute an indefinite integral using Gauss-Hermite quadrature of order `n`,
+    optionally shifting and scaling the quadrature points to focus on the
+    high-density region.
+
+    Parameters
+    ----------
+    func : callable
+        A Python function or method to integrate (must accept vector inputs).
+        For best results, `func` should approximately equal a Gaussian
+        density times a low-order polynomial.
+    n : int
+        Order of quadrature integration.
+    args : tuple, optional
+        Extra arguments to pass to `func`, if any.
+    mu_hat : float, optional
+        Location parameter, used to center the quadrature nodes near
+        the high-density region (e.g. the mean or mode if `func` is proportional
+        to a probability density).
+    sigma_hat : float, optional
+        Scale parameter, used to determine how much to spread out the quadrature
+        nodes (e.g. the standard deviation if `func` is proportional to a
+        probability density).
+    zero_nan : bool, optional
+        If `func(x)` returns `nan` (e.g. if a quadrature point falls outside the
+        domain of x), should this value be coerced to zero? Default is False.
+
+    Returns
+    -------
+    val : float
+        Gauss-Hermite quadrature approximation to the indefinite integral of
```

Comment:
Looks like you're right. Maybe the term I was looking for is "improper integral"? http://mathworld.wolfram.com/ImproperIntegral.html

Or maybe it's better just to say that it integrates from `-Inf` to `Inf`.